packages/salesforcedx-vscode-apex-replay-debugger/src/commands/apexExecutionOverlayActionCommand.ts

```diff
@@ -22,7 +23,6 @@ export interface ApexExecutionOverlaySuccessResult {
 export class ApexExecutionOverlayActionCommand extends BaseCommand {
   private readonly apiPath = 'services/data/v43.0/tooling/sobjects';
```

Comment:
Could you extract this into a constant? It would be very weird if this is the only one that uses 43.0 when everything else is updated later.